openhtf/core/diagnoses_lib.py

```diff
@@ -204,8 +203,8 @@ def _convert_result(self,
       return
     elif isinstance(diagnosis_or_diagnoses, Diagnosis):
       yield self._verify_and_fix_diagnosis(diagnosis_or_diagnoses, diagnoser)
-    elif (isinstance(diagnosis_or_diagnoses, six.string_types) or
-          not isinstance(diagnosis_or_diagnoses, collections_abc.Iterable)):
+    elif (isinstance(diagnosis_or_diagnoses, str) or
+          not isinstance(diagnosis_or_diagnoses, collections.abc.Iterable)):
```

Comment:
Consider importing this as 
from collections.abc import Iterable

...the same way as you'd import CamelCased types from the typing module.

Reply:
Done.